lib/getPostcssResult.js

```diff
@@ -19,7 +25,7 @@ module.exports = function (stylelint/*: stylelint$internalApi*/)/*: Promise<?Obj
     customSyntax?: string
   }*/ = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {}
 
-  const cached = stylelint._postcssResultCache.get(options.filePath)
+  const cached/*: Object*/ = stylelint._postcssResultCache.get(options.filePath)
```

Comment:
This should be a PostCSS Result, right — not just an Object? I'm not sure the Object annotation is very helpful, because it does not add any information. If you want to create a new `postcss$result` `type` declaration, that could be helpful!